protocols/kad/src/behaviour.rs

```diff
@@ -769,20 +892,24 @@ where
                                     NetworkBehaviourAction::GenerateEvent(event));
                             },
                             kbucket::InsertResult::Full => {
-                                debug!("Bucket full. Peer not added to routing table: {}", peer)
+                                debug!("Bucket full. Peer not added to routing table: {}", peer);
+                                let address = addresses.first().clone();
+                                self.queued_events.push_back(NetworkBehaviourAction::GenerateEvent(
+                                    KademliaEvent::PendingRoutablePeer { peer, address }
```

Comment:
Yes, this is a mistake. Thanks!
```suggestion
                                    KademliaEvent::RoutablePeer { peer, address }
```